kong-ngx-build

```diff
@@ -462,8 +466,14 @@ version_lt() {
   parse_version $2 version_b
 
   for index in "${!version_a[@]}"; do
-    [[ "${version_a[index]}" < "${version_b[index]}" ]] && return 0
-    [[ "${version_a[index]}" > "${version_b[index]}" ]] && return 1
+    if [[ ${version_a[index]} =~ ^[0-9]+$ ]]; then
+      [[ "${version_a[index]}" -lt "${version_b[index]}" ]] && return 0
+      [[ "${version_a[index]}" -gt "${version_b[index]}" ]] && return 1
+
+    else
+      [[ "${version_a[index]}" < "${version_b[index]}" ]] && return 0
+      [[ "${version_a[index]}" > "${version_b[index]}" ]] && return 1
+    fi
```

Comment:
This would still fail in a comparison like `openssl-1.1.99a openssl-1.1.100a`

Reply:
Thanks, fixed in the latest version of this branch